library/core/src/mem/maybe_uninit.rs

```diff
@@ -703,7 +700,7 @@ impl<T> MaybeUninit<T> {
     ///    // Reference to an uninitialized `Cell<bool>`: UB!
     /// }
     /// ```
-    #[unstable(feature = "maybe_uninit_ref", issue = "63568")]
+    #[stable(feature = "maybe_uninit_ref", since = "1.55.0")]
```

Comment:
Shouldn't this be 1.54?

Reply:
OTOH, it's release week, so this will probably land after the beta branches.

Reply:
#86234 landed already so it should be 1.55.0 IIUC.